hs/Response/Draw.hs

```diff
@@ -47,3 +46,12 @@ modePanel = H.div ! A.id "side-panel-wrap" $ do
     H.div ! A.id "finish-region" ! A.class_ "button" $ "finish (f)"
     H.div ! A.id "change-mode" ! A.class_ "mode" $ "SELECT/MOVE (m)"
     H.div ! A.id "erase-mode" ! A.class_ "mode" $ "ERASE (e)"
+    H.input ! A.id "area-of-study"
+            ! A.class_ "course-code"
+            ! A.name "course-code"
+            ! A.placeholder "Enter area of study."
+            ! A.autocomplete "off"
+            ! A.type_ "text"
+            ! A.size "30"
+    H.div ! A.id "submit-gid" ! A.class_ "button" $ "Submit"
```

Comment:
Based on the documentation: https://hackage.haskell.org/package/blaze-html-0.1/docs/Text-Blaze.html#2


From HTML 5 onwards, the user is able to specify custom data attributes.

An example:

 <p data-foo="bar">Hello.</p>
We support this in BlazeHtml using this funcion. The above fragment could be described using BlazeHtml with:

 p ! dataAttribute "foo" "bar" $ "Hello."